msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -38,7 +38,9 @@ The following table shows a list of Teams conversation update events, with links
 | members added   | membersAdded   | OnTeamsMembersAddedAsync   | [A member added](#team-members-added)   | All |
 | members removed | membersRemoved | OnTeamsMembersRemovedAsync | [A member was removed](#team-members-removed) | groupChat & team |
 | team renamed        | teamRenamed       | OnTeamsTeamRenamedAsync    | [A Team was renamed](#team-renamed)       | Team |
+| team deleted        | teamDeleted       | OnTeamsTeamDeletedAsync    | [A Team was deleted](#team-deleted)       | Team |
 | team archived        | teamArchived       | OnTeamsTeamArchivedAsync    | [A Team was archived](#team-archived)       | Team |
+| team unarchived        | teamUnarchived       | OnTeamsTeamUnarchivedAsync    | [A Team was unarchived](#team-unarchived)       | Team |
 | team restored        | teamRestored      | OnTeamsTeamRestoredAsync    | [A Team was renamed](#team-renamed)       | Team |
```

Comment:
The description here is "A team was renamed"

Reply:
fixed